src/sails_forth/client.clj

```diff
@@ -6,8 +6,10 @@
             [sails-forth.clojurify :as clj]))
 
 (defprotocol Cache
-  (put! [_ key value])
-  (get! [_ key]))
+  (^:spark/no-boot-spec-coverage
```

Comment:
The only thing I don't like about this metadatum is the specificity. It's a procedural marker - don't do this thing for these vars. How would you feel about, in the next iteration, a declarative marker - `:spark/impure` or similar? One that describes a general property of the fn which the tool could take advantage of?